FlashDevelop/Bin/Debug/Settings/Languages/Haxe.xml

```diff
@@ -39,9 +39,9 @@
 				<keyword key="1" class="haxe-additional-keywords" />
 				<keyword key="2" class="haxe-documentation-keywords" />
 				<keyword key="3" class="haxe-primary-keywords" />
-				<keyword key="5" class="haxe-additional-keywords3" />
```

Comment:
Only Haxe updated? And this will break on all existing config/theme files. We need to keep the old indexes.

We could force refresh configs but it still breaks on custom or standalone configs.